libstuff/SSignal.cpp

```diff
@@ -209,6 +210,9 @@ void _SSignal_StackTrace(int signum, siginfo_t *info, void *ucontext) {
                 SWARN("Stack depth is " << depth << " only logging first and last 20 frames.");
             }
 
+            // We mainly depend on syslog to investigate crashes, but when performance is real bad, sometimes we lose those logs.
+            // For cases like those, we'll also save the stack trace for the crash in a file.
+            int fd = creat(format("/tmp/bedrock_crash_{}.log", STimeNow()).c_str(), 0666);
```

Comment:
Can we not save important data in `/tmp` please? It is literally in the name; it's supposed to be for temporary files. Using `/var/lib/bedrock/` would be far preferable.

Reply:
Is that a real problem here? We should, in theory, rarely have to use this since the logs should be available in logsearch. we'll only need it if the es backlog is too high, and if that happens we'll be on an active fire and should be able to check the crash log immediately.